traveller/config.json

```diff
@@ -9,5 +9,6 @@
       "ACTIVE_FRONT_THEME": "blogus",
       "ACTIVE_BACK_THEME": "boogle",
       "CURRENCY": "MUR"
-    }
+    },
+    "UPLOADS_AUTOSERVE": false
```

Comment:
It should be in config.py

Reply:
Done 👍🏼

Reply:
I mean, you dont need to touch config.json. It needs to be in this file under base config

https://github.com/FlaskCon/traveller/blob/dev/traveller/config.py

Reply:
We are good to go else

Reply:
@Abdur-rahmaanJ Yes. That's the usual setup. We need to configure nginx on the staging server to serve the static files.